sqlmesh/core/engine_adapter/base.py

```diff
@@ -988,7 +1008,11 @@ def create_view(
         )
         if create_view_properties:
             for view_property in create_view_properties.expressions:
-                properties.append("expressions", view_property)
+                # Small hack to make sure SECURE goes at the beginning before materialized as required by Snowflake
```

Comment:
I guess consolidating the `materialized` key may be a bit more involved, and would probably lead us to eventually deprecate it, but did we conclude that we don't want to allow folks to set it using this new property?

Reply:
Yes. Materialized has special handling besides just setting a modified. For example, when deleting a materialized view we need to include the modifier as well. Also, in some engines materialized views need to be manually refreshed.

Reply:
Should be a test for this now and it raises an exception.